src/inngest/functions.ts

```
@@ -1,43 +1,169 @@
-import { Sandbox } from "@e2b/code-interpreter"
-import { openai as openai, createAgent } from "@inngest/agent-kit";
+import { Sandbox } from "@e2b/code-interpreter";
+import {
+  openai as openai,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)